### ISSUE TITLE
PSA for new adopters - g and b values are flipped

### ISSUE THREAD

**Issue by ConorKelleher** [bfred-it/get-canvas-pixel-color#1]
I realise this repo hasn't been updated in over 4 years, I'm mostly just putting this here in case any new users come along and have trouble with it.

There's a typo in the main file for setting up the returned object, resulting in the `g` and `b` values to be flipped.

In these lines of code:
https://github.com/bfred-it/get-canvas-pixel-color/blob/master/index.js#L19
the object is set up like so:

```javascript
pixel.r = pixel[0];
pixel.b = pixel[1];
pixel.g = pixel[2];
pixel.a = pixel[3];
```
The `b` value is coming from index 1 and `g` is coming from index 2, meaning the returned object (as well as the concatenated string) are wrong. This can be easily gotten around if you know the issue is there. Just letting people know the issue is there.

**Comment by fregante** [comment]
Oh god, thank you for this :D 

I'll fix it

**Comment by ConorKelleher** [comment]
No problem, happy to help out :) Thanks for the quick response

**Comment by fregante** [comment]
Thanks for opening an issue on an old repo :)

I published it as 2.0.0, let me know if it works correctly for you

**Comment by ConorKelleher** [comment]
Updated and it's working great now. Thanks for the quick turnaround 👍
